src/webui/static/app/epggrab.js

```diff
@@ -27,7 +27,7 @@ tvheadend.epggrab_base = function(panel, index) {
         labelWidth: 200,
         tbar: [triggerButton],
         help: function() {
-            new tvheadend.help(_('EPG Grab Configuration'), 'config_epggrab.html');
+            new tvheadend.help(_('EPG grabber configuration'), 'config_epggrab.html');
```

Comment:
Is it the configuration of the grabber, of of grabbing in general? If the latter, then "EPG grab" is probably correct vs "EPG grabber" - or even "EPG source" if you want to avoid it entirely